code-studio/src/js/components/playzone.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * @overview React for the End-of-Stage Experience
+ */
+/* global React */
+var color = require('@cdo/apps/color');
+
+export class PlayZone extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.links = [{
+      href: "/projects/artist/new",
+      img: "/shared/images/courses/logo_tall_artist.jpg",
+      title: this.props.i18n.t('components.playzone.artist_title'),
+      className: "artist",
+      description: this.props.i18n.t('components.playzone.artist_description'),
+    }, {
+      href: "/projects/playlab/new",
+      img: "/shared/images/courses/logo_tall_playlab.jpg",
+      title: this.props.i18n.t('components.playzone.playlab_title'),
+      className: "playlab",
+      description: this.props.i18n.t('components.playzone.playlab_description'),
+    }, {
+      href: "/projects",
+      img: "/shared/images/courses/logo_tall_applab.jpg",
+      title: this.props.i18n.t('components.playzone.projects_title'),
+      className: "projects",
+      description: this.props.i18n.t('components.playzone.projects_description'),
+    }];
+
+    this.styles = {
+      container: {margin: '20px'},
+      primaryHeader: {fontSize: '200%'},
+      secondaryHeader: {color: color.charcoal},
+      courseblockContainer: {width: '720px', paddingTop: '20px'},
+      continueButton: {marginTop: '20px', marginRight: 0}
+    };
+  }
+
+  render() {
+    return (
+      <div style={this.styles.container}>
+        <h1 style={this.styles.primaryHeader}>{this.props.i18n.t('components.playzone.primary_header', {stageName: this.props.stageName})}</h1>
```

Comment:
I didn't think so at first either, but it turns out they do! https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/layouts/_javascript_strings.haml#L118